vedtekter/README.md

```diff
@@ -89,6 +92,8 @@ Ledelsen i TD heter styret og består av
 
 Styret og medlemmene avgjør selv hvorvidt det skal være en eller to teknisk ansvarlige når det nye styret velges på årsmøtet.
 
+6. én kommmunikasjonsansvarlig
```

Comment:
Tror jeg ville plassert denne over teksten for å spesifisere at styret og medlemmene selv bestemmer om det skal være én eller to teknisk ansvarlige